### ISSUE TITLE
Need hbase-site.xml and kafka configuration files for this api

### ISSUE THREAD

**Issue by neer3112** [mravi/hbase-cdc-kafka#3]
Hi Ravi,

Am new to HBASE and Kafka, need help, Can you please share the hbase-site.xml and kafka config files needed to use this connector (hbase-connect-kafka) with the required values.

Thanks,
Neeraj

**Comment by ghost** [comment]
yes, I need that too. Can you plz share them to us? God bless you!
